scripts/run.bash

```
@@ -37,56 +37,189 @@ join_by() {
+
+run_states_for_usb() {
+  local states_for_dom0_list=(
+    "sys-usb.dom0.setup"
+    "sys-usb.tpl.at-dom0.create-tpl"
+    "sys-usb.tpl.at-dom0.create-tpl-dvm"
+    "sys-usb.tpl.at-dom0.set-appmenu"
+    "sys-usb.qube.at-dom0.create"
+  )
+  local states_for_dom0
+  states_for_dom0="$(join_by ',' "${states_for_dom0_list[@]}")"
+  sudo qubesctl --show-output --targets=dom0 state.apply "$(join_by ',' "${states_for_dom0}" pillar="${data}"
```

Shellcheck (0.10.0)

[info] The mentioned syntax error was in this double quoted string.

(SC1009)